src/Conference/ConferenceDetails.jsx

```diff
@@ -74,6 +76,15 @@ const ConferenceDetails = () => {
   const { name, topic, dateTime, city, description } = conference || {};
   const date = new Date(dateTime);
 
+  const deleteAConference = async id => {
+    try {
+      await deleteConference(id);
+      history.push("/");
+    } catch (error) {
+      console.log(error);
```

Comment:
Just out of interest: Are you planning on doing something in the future if there's an error like like a notification?

Reply:
Next ticket is a toast component which might be used for this? currently we're just console logging here as we haven't really figured out how we want to do error handling.